docs/i18n/espanol/how-to-work-on-coding-challenges.md

```diff
@@ -27,55 +27,59 @@ Antes de trabajar en el plan de estudios, necesitarás configurar algunas herram
 
   [![Abrir en Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/freeCodeCamp/freeCodeCamp)
 
-- Editar los archivos de la interfaz de GitHub haciendo clic en el icono del lápiz del archivo correspondiente. Aunque esta es la manera más rápida, **no se recomienda**, ya que no puedes probar tus cambios en GitHub. If our maintainers conclude that the changes you made need to be tested locally, you would need to follow the methods above instead.
+- Editar los archivos de la interfaz de GitHub haciendo clic en el icono del lápiz del archivo correspondiente. Aunque esta es la manera más rápida, **no se recomienda**, ya que no puedes probar tus cambios en GitHub. Si nuestros administradores concluyen que los cambios que hiciste necesitan ser probados de forma local, entonces necesitarás seguir los métodos de encima.
 
-### How to work on practice projects
+### Cómo trabajar en proyectos de práctica
 
-The practice projects have some additional tooling to help create new projects and steps. To read more, see [these docs](how-to-work-on-practice-projects.md)
+Los proyectos de práctica tienen algunas herramientas adicionales para ayudar a crear nuevos proyectos y pasos. Para leer más, consulta [ estos documentos ](how-to-work-on-practice-projects.md)
 
 ## Plantilla de desafío
 
 ````md
+Esto no ha sido finalizado completamente, pero lo siguiente debería estar cerca de la estructura final:
+
+````mdx
+
 ---
-id: Unique identifier (alphanumerical, MongoDB_id)
-title: 'Challenge Title'
-challengeType: Integer, defined in `client/utils/challenge-types.js`
-videoUrl: 'url of video explanation'
+id: Identificador único (alfanumérico, MongoDB_id)
+title: 'Título del Desafío'
+challengeType: Integer, definido en `client/utils/challengeTypes.js`
```

Comment:
Is this ok? Will the next round of approvals/Crowdin-downloads fix this?

Reply:
What's the concern here? It makes sense to translate this because it's descriptive content rather than actual code.

Reply:
The original within backticks is the correct file name: `challenge-types.ts`. This "translation" leads to an incorrect reference to a non-existent file

Reply:
Oh I see

This is why we require 2 reviews 😅 

Will fix.